sql/core/src/main/scala/org/apache/spark/sql/execution/subquery.scala

```diff
@@ -147,6 +147,17 @@ case class InSubqueryExec(
     }
   }
 
+  // Visible for testing
+  private[sql] def predicate: Predicate = {
```

Comment:
I think you can remove `private[sql]` because `execution` package is already private.

Reply:
There is no hard limit, you mean just `def predicate` ?

Reply:
Yup.